package.json

```diff
@@ -8,11 +8,13 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "chart.js": "^2.9.4",
     "axios": "^0.21.0",
     "node-sass": "^4.14.1",
     "material-auto-rotating-carousel": "^3.0.2",
     "moment": "^2.29.1",
     "react": "^17.0.0",
+    "react-chartjs-2": "^2.10.0",
```

Comment:
what's this for ?

Reply:
Ah oui, j'ai installé ça pour la graph-chart que j'etait en train de faire mais pas encore fini @ComicScrip

Reply:
Mais c'est un autre composent donc je crois que je les ai installé dans la wrong branch? je les annule?

Reply:
@KarenOrduna yes you can remove them, and we'll add those deps when they are really needed :)

Reply:
AAH I got it. C'est un package.json global! It's for a new component Graph which I haven't included yet because it's 1/3 done.

Reply:
okay done :D  @ComicScrip